DuckDB.NET.Data/DuckDBDataReader.cs

```diff
@@ -249,5 +239,12 @@ public override void Close()
 
             closed = true;
         }
+
+        protected override void Dispose(bool disposing)
```

Comment:
It's not necessary to implement `Dispose` because `Dispose` calls `Close` and any cleanup can be done there.